packages/fast-components-react-msft/src/context-menu/examples.data.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import { ContextMenu, ContextMenuHandledProps } from "./index";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import schema from "@microsoft/fast-components-react-base/dist/context-menu/context-menu.schema.json";
+import Documentation from "./.tmp/documentation";
+
+const divider: any = {
+    id: "divider",
+    props: {
+        jssStyleSheet: {
+            divider: {
+                margin: "0 4px",
+                opacity: ".5",
+            },
+        },
+    },
+};
+export default {
+    name: "Context menu",
+    component: ContextMenu,
+    schema: schema as any,
+    documentation: <Documentation />,
+    detailData: {
+        children: [
+            {
+                id: "context-menu-item",
```

Comment:
does it matter than all these ids are the same?

Reply:
Left over from when they were required, I'll just remove.

Reply:
nevermind - these id's all need to be the same. These inform the form-generator.